config/templates/socialaccount/login.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load socialaccount %}
+
+{% block content %}
+<div class="flex items-center justify-center transition-colors duration-300">
+    <div class="w-full max-w-md p-6 bg-white dark:bg-gray-800 rounded-lg shadow-md">
+        <h2 class="text-2xl font-bold text-center text-gray-800 dark:text-white">Sign in with Google</h2>
+        <p class="text-gray-600 dark:text-gray-400 text-center mt-2">
+            You're about to sign in using your Google account.
+        </p>
+
+        <div class="mt-6">
+            <form method="post">
+                {% csrf_token %}
+                <button type="submit"
+                class="w-full flex items-center justify-center bg-blue-600 hover:bg-blue-700 text-white font-medium py-3 px-4 rounded-lg shadow transition duration-300">
```

Comment:
I agree it looks good! I might suggest using the primary button style, since the default Flowbite blues aren't used on the site:

```suggestion
                class="w-full flex items-center justify-center btn btn-outline btn-primary font-medium py-3 px-4 rounded-lg shadow transition duration-300">
```